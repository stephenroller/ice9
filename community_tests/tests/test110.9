# var i : int [6], d ; (missing colon & typeid)			fail

var i : int [6], d ;