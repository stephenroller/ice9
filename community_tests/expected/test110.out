line 11: too many array dereferences in l-value